libc/string.c

```diff
@@ -0,0 +1,184 @@
+/*
+ * Phoenix-RTOS
+ *
+ * libc-tests
+ *
+ * Testing string.h functions
+ *
+ * Copyright 2021 Phoenix Systems
+ * Author: Mateusz Niewiadomski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <string.h>
+
+#include <unity_fixture.h>
+
+/* fixtures for strlcpy */
+char source[5];
+char dest[5];
+char tmplt[5];
+
+/* fixtures for strlcat */
+char source1[4];
+char source2[6];
+char buffer[12];
+
+/* common fixtures */
+int i;
```

Comment:
Common iterator name for global variable - I'm not sure if I like it (I think I don't). Maybe `retval`?

Reply:
Or maybe it should be a local variable (along with the name change)

Reply:
Variable left as global. Name changed to `retval` as suggested. Thanks!